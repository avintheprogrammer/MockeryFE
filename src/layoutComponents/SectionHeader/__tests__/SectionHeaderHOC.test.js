import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import { SectionHeader } from '../index';
import mock from '../__mocks__/SectionHeaderHOC.mock';

configure({ adapter: new Adapter() });

test('Section Header renders correctly', () => {
  const tree = shallow(<SectionHeader data={mock} />);
  expect(toJson(tree)).toMatchSnapshot();
});

test('Render the banner if data is valid.', () => {
  const wrapper = shallow(<SectionHeader data={mock} />);
  expect(wrapper.text()).toBe('<Banner />');
});
