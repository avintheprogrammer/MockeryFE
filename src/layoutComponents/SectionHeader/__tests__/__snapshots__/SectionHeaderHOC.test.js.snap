// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Header renders correctly 1`] = `
<Banner
  backgroundType="color"
  brand="mockery"
  data={
    Object {
      "section": Object {
        "title": "the section",
      },
      "title": "hello",
    }
  }
  dataId={0}
  eyebrowType="text"
  sectionColor=""
/>
`;
